### ISSUE TITLE
no matching clause for '#%linklet-primitive

### ISSUE THREAD

**Issue by ribrdb** [vishesh/racketscript#164]
I'm getting this error
```
[info][expand] /Applications/Racket v7.6/share/pkgs/draw-lib/racket/draw/unsafe/callback.rkt
[info][absyn] callback
[warning] Implementation of identifier break-enabled-key not found in module #%paramz!
[warning] Implementation of identifier unsafe-undefined not found in module #%unsafe!
[warning] Implementation of identifier unsafe-undefined not found in module #%unsafe!
[info][il] callback
[debug] Processing library collection links at: /Users/ribrdb/Library/Racket/7.6/links.rktd
[debug] Processing library collection links at: /Applications/Racket v7.6/collects/../share/links.rktd
match: no matching clause for '#%linklet-primitive
  location...:
   /Users/ribrdb/Library/Racket/7.6/pkgs/racketscript-compiler/racketscript/compiler/config.rkt:79:18
  context...:
   /Applications/Racket v7.6/collects/racket/match/runtime.rkt:24:0: match:error
   /Users/ribrdb/Library/Racket/7.6/pkgs/racketscript-compiler/racketscript/compiler/config.rkt:78:0: jsruntime-module-path
   /Users/ribrdb/Library/Racket/7.6/pkgs/racketscript-compiler/racketscript/compiler/transform.rkt:87:4: for-loop
   /Users/ribrdb/Library/Racket/7.6/pkgs/racketscript-compiler/racketscript/compiler/transform.rkt:58:0: absyn-module->il
   /Users/ribrdb/Library/Racket/7.6/pkgs/racketscript-compiler/racketscript/compiler/main.rkt:190:0: absyn-module->il*
   /Users/ribrdb/Library/Racket/7.6/pkgs/racketscript-compiler/racketscript/compiler/main.rkt:274:2: loop
   (submod "/Users/ribrdb/Library/Racket/7.6/pkgs/racketscript-compiler/racketscript/compiler/main.rkt" main): [running body]
   temp35_0
   for-loop
   run-module-instance!
```

**Comment by stchang** [comment]
Hi. What is the example that produces the error?

**Comment by vishesh** [comment]
Should fix this issue by #181.
